vimeo-networking/build.gradle

```diff
@@ -67,19 +67,50 @@ artifacts {
     archives sourcesJar, javadocJar
 }
 
+// Create the pom configuration:
+def pomConfig = {
+    licenses {
+        license {
+            name "MIT License"
+            url "http://www.opensource.org/licenses/mit-license.php"
+            distribution "repo"
+        }
+    }
+    developers {
+        developer {
+            id "vimeo"
+            name "Vimeo Mobile"
+            email "mobileops@vimeo.com"
+            organisation "Vimeo"
+            organisationUrl "https://vimeo.com"
+        }
+    }
+    
+    scm {
+        connection "scm:git:git://github.com/vimeo/vimeo-networking-java.git"
+        developerConnection "scm:git:ssh://github.com:vimeo/vimeo-networking-java.git"
+        url "https://github.com/vimeo/vimeo-networking-java"
+    }
+}
+
+// Create the publication with the pom configuration:
 // Requires apply plugin: maven-publish
 publishing {
     publications {
-        mavenJava(MavenPublication) {
-            // We dont have any complex artifacts, so let's just reference the fact that we
-            // rely on plugin java
+        MyPublication(MavenPublication) {
             from components.java
+            artifact sourcesJar
+            artifact javadocJar
             groupId project.group
             artifactId 'vimeo-networking'
             version project.version
-
-            artifact sourcesJar
-            artifact javadocJar
+            pom.withXml {
+                def root = asNode()
+                root.appendNode('description', 'vimeo-networking is a Java networking library used for interacting with the Vimeo API.')
+                root.appendNode('name', 'vimeo-networking')
+                root.appendNode('url', 'https://vimeo.com')
```

Comment:
again maybe point this to github, thats what devs are more likely to find relevant